c/meterpreter/README.md

```diff
@@ -1,59 +1,121 @@
+        "Microsoft.VisualStudio.Component.NuGet",
+        "Microsoft.VisualStudio.Component.Roslyn.Compiler",
+        "Microsoft.Component.MSBuild",
+        "Microsoft.VisualStudio.Component.TextTemplating",
+        "Microsoft.VisualStudio.Component.IntelliCode",
+        "Microsoft.VisualStudio.Component.VC.CoreIde",
+        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.Graphics.Tools",
+        "Microsoft.VisualStudio.Component.VC.DiagnosticTools",
+        "Microsoft.VisualStudio.Component.Windows10SDK.18362",
+        "Microsoft.VisualStudio.Component.Debugger.JustInTime",
+        "Microsoft.VisualStudio.Component.VC.Redist.14.Latest",
+        "Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core",
+        "Microsoft.VisualStudio.Component.VC.CMake.Project",
+        "Microsoft.VisualStudio.Component.VC.ATL",
+        "Microsoft.VisualStudio.Component.VC.ASAN",
+        "Microsoft.Component.VC.Runtime.UCRTSDK",
+        "Microsoft.VisualStudio.Workload.NativeDesktop",
+        "Microsoft.VisualStudio.Component.WinXP"
+    ]
+}
+  ```
+3. Install VS with all the required components by running the following command in Powershell:
+  ```
+choco install visualstudio2019community -y --package-parameters "--config Path\To\Your\.vsconfig"
+  ```
+
+Note: A copy of this file is located in this repository under `c/meterpreter/vs-config/vs2019.vsconfig`.
+
+### VS 2017
+Building with VS2017 works with any version, including community. If you have an installation already, just make sure you have the following extra bits installed:
+
+* Under `Workloads`:
+    * `Desktop Development with C++`
+* Under `Individual Components`:
+    * `Windows XP support for C++`
+
+If you don't have an installation ready, follow these steps:
+
+1. Install [Chocolatey](https://chocolatey.org).
+2. Create a file called `.vsconfig` somewhere on disk with the following contents:
+  ```
+{
+    "version": "1.0",
+    "components": [
+        "Microsoft.VisualStudio.Workload.NativeDesktop",
+        "microsoft.visualstudio.component.debugger.justintime",
+        "microsoft.visualstudio.component.vc.diagnostictools",
+        "microsoft.visualstudio.component.vc.cmake.project",
+        "microsoft.visualstudio.component.vc.atl",
+        "microsoft.visualstudio.componentgroup.nativedesktop.winxp"
+    ]
+}
+  ```
+3. Install VS with all the required components by running the following command in Powershell:
+  ```
+choco install visualstudio2017community -y --package-parameters "--config Path\To\Your\.vsconfig"
+  ```
+
+Note: A copy of this file is located in this repository under `c/meterpreter/vs-config/vs2017.vsconfig`.
+
+### VS 2013
+Download and install the `Visual Studio Express 2013 for Windows Desktop` edition. It is important that you use _this exact version_. To do this with Chocolatey, run the following:
+
+```
+choco install vs2013expressdesktop5 -y
```

Comment:
```suggestion
choco install visualstudioexpress2013windowsdesktop -y
```

An existing approved package that meets build requirements.

Build result from `rake win_prep` on commit fda53ff
```
Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:04:44.43
```